_oasis

```diff
@@ -22,6 +25,7 @@ Library "cephes"
   BuildTools:       cephes_bindgen
 
 Executable "cephes_bindgen"
+  Build$:           flag(tests)
```

Comment:
`cephes_bindgen` is part of the build. It generates `cephes_generated.ml` and `cephes_stubs.c` that are used as `cephes.ml`. Shouldn't (or can) `oasis2opam` recognize that with the `BuildTools` declaration in the Library.

I'll take your point about wastefully building something. But I assume that `opam` wipes away the build directory after it installs a package, so there aren't errant unused generated files lying around.